src/components/common/FileUploadPreview.vue

```diff
@@ -88,25 +57,47 @@ export default class FileUploadPreview extends Mixins(DocumentMixin) {
 
   /** Programmatically opens the file selection dialog. Can be called externally. */
   public clickFileInput (): void {
+    // first clear any existing file
+    this.fileUpload = null
     // find the find input element in THIS component (not globally)
     const element = this.$el.querySelector('#file-input') as HTMLElement
+    // click the file input element to open the file selection dialog
     element.click()
   }
 
-  // Note: the validation is done this way as opposed to being all defined in the validation rules(fileUploadRules)
-  //  above because Vuetify does not support async validation out of the box.  This was needed to work
-  //  around this limitation.  vuelidate does support this but this would mean introducing a component that
-  //  is using a different validation approach or updating all components to use vuelidate.  Have decided
-  //  to do this for the time being.
+  /**
+   * Validates the file input. Includes "rules" here because Vuetify does not support async validation.
+   * @param file The file to validate.
+   */
   private async validateFileInput (file: File): Promise<boolean> {
-    // clear any existing error messages
-    this.setCustomErrorMessage(null)
+    // clear any previous error state
+    this.setCustomErrorMessage('')
```

Comment:
So this is kind of key. By setting or clearing this v-file-input prop, we can put it in error state or not.

In a nutshell, it looks and works exactly like before... except setCustomErrorMessage() also syncs to the parent so that the parent can display the error message themselves.

Unfortunately, using error messages in the rules, we could not emit the error message.